api/v1alpha1/backstage_types.go

```diff
@@ -19,15 +19,15 @@ import (
 )
 
 const (
-	RuntimeConditionRunning string = "RuntimeRunning"
-	RuntimeConditionSynced  string = "RuntimeSyncedWithConfig"
-	RouteSynced             string = "RouteSynced"
-	LocalDbSynced           string = "LocalDbSynced"
-	SyncOK                  string = "SyncOK"
-	SyncFailed              string = "SyncFailed"
-	Deleted                 string = "Deleted"
-	EnvPostGresImage        string = "RELATED_IMAGE_postgresql"
-	EnvBackstageImage       string = "RELATED_IMAGE_backstage"
+	//RuntimeConditionRunning string = "RuntimeRunning"
+	ConditionSynced        string = "Synced"
```

Comment:
"Synced" is confusing in all of those 2 IMO, in fact, it states that runtime objects are deployed/working/running... 

Note: Initially "Synced" indicated that runtime is in sync with CR (CR "owned" runtime).

Reply:
> "Synced" is confusing in all of those 3 IMO, in fact, it states that runtime objects are deployed/working/running...
> 
> Note: Initially "Synced" indicated that runtime is in sync with CR (CR "owned" runtime).
I borrowed the concept "Synced" from the Argo CD Operator which has been widely used in the community.
3 status conditions are used because it allows users whether the localdb has been successfully "synced" and whether the route has been created along with the route host information. It is hard to use a single condition type as the later status overrides the previous one.
In addition, it is pretty common to use separate condition types, for instance, in the k8s deployment object, we can see that both "Progressing" which indicates the deployment status and "Available" that is to indicate the runtime status are used.

Reply:
"Sync" does make sense in GitOps (ArgoCD) case where you have desired state remotely (git repository) which is not our case.

I am not debating number of conditionals in general, I mean is seems to me it is too much for our case, I'd prefer to start smaller until necessity of more is proven

Reply:
I simplified the solution. Please take a look again.